### PR TITLE
[integration-guides] Separate out integration guides, add exchange guide

### PR DESCRIPTION
**Pull request by gregnazario**
### Checklist

- Do all Lints pass?
  - [ ] Have you ran `pnpm spellcheck`?
  - [ ] Have you ran `pnpm fmt`?
  - [ ] Have you ran `pnpm lint`?

### CODE REVIEW

File: apps/nextra/pages/en/build/guides/exchanges.mdx [hunk 1]
```diff
@@ -0,0 +1,1180 @@
...
+only talk about supporting `primary_fungible_store` for fungible assets.
+
+More info can be found at: [Fungible Asset Standard](/build/smart-contracts/fungible-assets.mdx)
+
+## Retrieving Balances
+
+Retrieving current balances for assets are different for each standard.  Integration is considered complete when it can handle both.
+
+Balances are always returned in their subunits.  For example, `APT` is returned in `octas` (1e-8 APT).  So, when an API
+returns a balance of `100000000`, this is `1 APT`.  If it returns `100` this is `0.000001 APT`.
```
Comment (lightmark) [inline apps/nextra/pages/en/build/guides/exchanges.mdx:108]: add `,`

File: apps/nextra/pages/en/build/guides/exchanges.mdx [hunk 2]
```diff
@@ -0,0 +1,1180 @@
...
+        }
+      }
+    },
+    "type": "write_resource"
+  }],
+```
+</details>
+
+3. Events are the events that were emitted by the transaction.  In this case, we
+only care about the `0x1::coin::deposit` and `0x1::coin::withdraw` events.
```
Comment (lightmark) [inline apps/nextra/pages/en/build/guides/exchanges.mdx:328]: do we still want this or v2 events?

File: apps/nextra/pages/en/build/guides/exchanges.mdx [hunk 3]
```diff
@@ -0,0 +1,1180 @@
...
+An example: https://api.mainnet.aptoslabs.com/v1/transactions/by_version/1750174030
+
+There are a few steps when tracking fungible assets:
+
+1. There will be two types of events for fungible assets.  `0x1::fungible_asset::Deposit` and `0x1::fungible_asset::Withdraw`.
+
+`Withdraw` events are similar to the coin events, where the balance will decrease by the amount in the `data.amount` field.
+And similarly `Deposit` events will increase the balance by the amount in the `data.amount` field.
+
+Note that, I've omitted the sequence number, and GUID fields, as they do not apply to these events.
```
Comment (lightmark) [inline apps/nextra/pages/en/build/guides/exchanges.mdx:669]: it's module events [link]

File: apps/nextra/pages/en/build/guides/exchanges.mdx [hunk 4]
```diff
@@ -0,0 +1,1180 @@
...
+
+For coins migrated to fungible assets, it is just simply tracking of the two above.
+A coin migrated to a fungible asset will have both the coin store changes and the
+primary fungible asset store changes.  The amounts would need to be aggregated
+together, and otherwise, handled as a coin.
+
+The Fungible asset metadata address is the hash of the coin type and 0xA
+
+```
+address = sha3_256(0xA | coin_type)
```
Comment (lightmark) [inline apps/nextra/pages/en/build/guides/exchanges.mdx:767]: sha3_256(0xA | coin_type | 0xfe)

also, APT is an exception.

File: apps/nextra/pages/en/build/guides/exchanges.mdx [hunk 5]
```diff
@@ -0,0 +1,1180 @@
...
+    ],
+    "timestamp": "1724196287102837",
+    "type": "user_transaction"
+  }
+  ```
+</details>
+
+## Transferring Assets
+
+### Coin (or migrated coin) Transfers
```
Comment (lightmark) [inline apps/nextra/pages/en/build/guides/exchanges.mdx:1113]: could we highlight you must use coin::transfer for migrated coins and pfs::transfer to pure FA.
And coin are all migrated so there is no `Coin`, only `Migrated Coin`.

**Review by lightmark — APPROVED** [review]

File: apps/nextra/pages/en/build/apis/faucet-api.mdx [hunk 6]
```diff
@@ -6,66 +6,71 @@ title: "Faucet API"
...
+Most wallets, such as [Petra](https://aptosfoundation.org/ecosystem/project/petra) or [Pontem](https://aptosfoundation.org/ecosystem/project/pontem-wallet)
+will have a faucet button for devnet and testnet. See full list of [Aptos Wallets](https://aptosfoundation.org/ecosystem/projects/wallets).
 
-1. Generate a client from
-the [OpenAPI spec](https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos-faucet/doc/spec.yaml).
-2. Call the faucet on your own.
+### Using the faucet in the Aptos CLI
 
-For the latter, you will want to build a query similar to this:
+Once you've [set up your CLI](../cli/setup-cli.mdx), you can simply call fund-with-faucet.  The amount used is in Octas (1 APT = 1,000,000,000 Octas).
```
[the remark this answers is not in the document]
Comment (gregnazario) [reply apps/nextra/pages/en/build/apis/faucet-api.mdx:26]: woops wrong number of 0s, maybe copilot did this

**Pull request merged**